stablehlo/dialect/StablehloOps.td

```diff
@@ -1797,19 +1797,11 @@ def StableHLO_ConcatenateOp : StableHLO_ShapedInterfaceOp<"concatenate",
      DeclareOpInterfaceMethods<InferTypeOpInterface>]> {
   let summary = "XLA's concatenate op";
   let description = [{
-     Concatenates a set of tensors along the specified dimension.
-
-     See https://www.tensorflow.org/xla/operation_semantics#concatenate.
-
-     Example:
-
-     ```mlir
-     %0 = stablehlo.concatenate %arg0, %arg1, dim = 1 : (tensor<4x1xf32>, tensor<4x2xf32>) -> tensor<4x3xf32>
-     ```
```

Comment:
Let's wait for @GleasonK to chime in on what we should do with the pretty syntax. Also see https://github.com/openxla/stablehlo/pull/250#discussion_r987523267.

Reply:
(Going through my notifications now, pasting the resolution from #250 for easy access)

> How about instead of removing everything from the description, we keep: 1) one sentence that describes the op, 2) pretty syntax, 3) link to spec?